src/DotNest.Core.SDK.Web/DotNest.Core.SDK.Web.csproj

```diff
@@ -33,8 +33,8 @@
     <PackageReference Include="Lombiq.Hosting.Tenants.IdleTenantManagement" Version="3.0.0" />
     <PackageReference Include="Lombiq.Privacy" Version="4.0.0" />
     <PackageReference Include="Lombiq.SetupExtensions" Version="3.0.0" />
-    <PackageReference Include="Lombiq.Hosting.MediaTheme" Version="2.0.0" />
-    <PackageReference Include="Lombiq.Hosting.MediaTheme.Bridge" Version="2.0.0" />
+    <PackageReference Include="Lombiq.Hosting.MediaTheme" Version="2.1.2-alpha.0.nest-407-7" />
```

Comment:
Shouldn't this (and the other MediaTheme packages) be `3.0.0-alpha.0.nest-407` as in OSOCE?

Reply:
You are right

Reply:
This was forgotten after the huge updates